doc/guide/bindings.hpp

```diff
@@ -408,12 +449,39 @@ Python binding output (snippet):
 
   >>> output = program(input=x)
   >>> model = output['output_model']
+
+Julia binding output (snippet):
+
+  For example, to train a model on the dataset `x` and save the output model to
+  `model`, the following command can be used:
+
+  julia> model = program(input=x)
+
+Go binding output (snippet):
+
+  For example, to train a model on the dataset "x" and save the output model to
+  "model", the following command can be used:
+
+    // Initialize optional parameters for Program().
+    param := mlpack.ProgramOptions()
+    param.Input = x
+
+    model := mlpack.Program(param)
 @endcode
 
 @code
 Input C++ (full program, 'random_numbers_main.cpp'):
 
-  PROGRAM_INFO("Random Numbers", "This program generates random numbers with a "
+// Program Name.
+BINDING_PNAME("Random Numbers");
+
+// Short description.
+BINDING_SHORT_DESC(
+      "An implementation of Random Numbers");
+
+// Long description.
+BINDING_LONG_DESC(
+      "This program generates random numbers with a "
```

Comment:
```suggestion
  // Program Name.
  BINDING_PNAME("Random Numbers");

  // Short description.
  BINDING_SHORT_DESC("An implementation of Random Numbers");

  // Long description.
  BINDING_LONG_DESC(
      "This program generates random numbers with a "
```
Just some quick fixes to match the indentation that was previously used.  I guess, if you wanted, we could also "de-indent" all of it.

Reply:
This looks great, thank you for the suggestion. :+1: